mongodbatlas/serverless_instances.go

```diff
@@ -57,7 +57,7 @@ type ServerlessCreateRequestParams struct {
 type ServerlessUpdateRequestParams struct {
 	ServerlessBackupOptions      *ServerlessBackupOptions `json:"serverlessBackupOptions"`
 	TerminationProtectionEnabled *bool                    `json:"terminationProtectionEnabled,omitempty"`
-	Tag                          []*Tag                   `json:"tags,omitempty"`
+	Tag                          []*Tag                   `json:"tags"`
```

Comment:
this is a patch, you probably want 
```suggestion
	Tag                          *[]*Tag                   `json:"tags"`
```

Or you'll delete the tags when only updating `TerminationProtectionEnabled`

Reply:
Good shout, I'll make that commit and add this to tags in ServerlessCreateRequestParams and Cluster clusters.go too